### DISCUSSION TITLE
Bölünebilirlik

### DISCUSSION THREAD

**Discussion by KayraMertAydin** [sonsuzus/sonsuzus#419] [Soru Cevap]
Negatif olmayan tamsayılar içeren N boyutunda bir A dizisi sağlanır. Göreviniz, tüm N sayıların son rakamı seçilerek oluşturulan sayının 10'a bölünüp bölünemeyeceğini belirlemektir.
.

Not: Daha fazla açıklama için örnek açıklama bölümünü inceleyin.

Giriş formatı

İlk satır: A dizisinin boyutunu belirten tek bir tam sayı N
İkinci satır: N adet boşlukla ayrılmış tamsayı.
Çıkış biçimi

Sayı 10'a bölünüyorsa ''Evet'' yazdırın,
aksi takdirde ''Hayır'' yazdırın

Örnekler: 

1 ≤ N ≤ 10⁵
0 ≤ A[i] ≤ 10⁵

**Comment by KayraMertAydin** [comment]
https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/divisible-or-not-81b86ad7/

**Comment by Erva-Naz** [comment]
#include<bits/stdc++.h>
#include <stdio.h>
    using namespace std;
    int main()
    {
		int n;

		cin>>n;
		int a[n], b[n];
		for(int i=0;i<n;i++) {
			cin>>a[i];
		}

		if(a[n-1]%10 ==0){
			cout<<"Yes"<<endl;
		}

		else {
			cout <<"No"<<endl;
		}
		}
